Added translated readme file (brazilian portuguese) (#2234)
mgba-emu/mgba: README_PT_BR.md

<h2>Características</h2>

<ul>
<li>Suporte altamente preciso do hardware do Game Boy Advance<a href="#missing"><sup>[1]</sup></a>.</li>
<li>Suporte pro hardware do Game Boy/Game Boy Color.</li>
<li>Emulação rápida. Conhecido por executar na velocidade máxima até mesmo nos hardwares low-end tais como netbooks.</li>
<li>Portes do Qt e SDL pra um frontend peso-pesado e um peso-leve.</li>
<li>Suporte pro cabo do link local (do mesmo computador).</li>
<li>Detecção do tipo do save até mesmo pro tamanho da memória do flash<a href="#flashdetect"><sup>[2]</sup></a>.</li>
<li>Suporte pra cartuchos com sensores de movimento e pro rumble (só usável com controles do jogo).</li>
<li>Suporte pra relógio em tempo real até mesmo sem configuração.</li>
<li>Suporte pra sensor solar pros jogos Boktai.</li>
<li>Suporte pra Câmera do Game Boy e pra Impressora do Game Boy.</li>
<li>Uma implementação embutida da BIOS e a habilidade de carregar arquivos externos da BIOS.</li>
<li>Suporte pra turbo/avanço rápido segurando o Tab.</li>
<li>Rebobinação segurando o Backquote.</li>
<li>Frameskip, configurável até 10.</li>
<li>Suporte pra screenshot.</li>
<li>Suporte pra códigos de trapaça.</li>

> <li>9 slots pra savestates. Os savestates são também visíveis como screenshots.</li>

Reviewer: ```suggestion
<li>9 slots para savestates. Os savestates são também visíveis como screenshots.</li>
```

<p>Os seguintes mappers são suportados parcialmente:</p>

<ul>
<li>MBC6 (suporte pra gravação da memória flash ausente)</li>
<li>MMM01</li>
<li>Pocket Cam</li>
<li>TAMA5 (Suporte pro RTC ausente)</li>
<li>HuC-1 (Suporte pro IR ausente)</li>
<li>HuC-3 (Suporte pro RTC e IR ausente)</li>
</ul>

<h3>Características planejadas</h3>

<ul>
<li>Suporte pro cabo do link do multiplayer em rede.</li>
<li>Suporte pro cabo do link do barramento do Dolphin/JOY.</li>
<li>Mixagem de áudio do MP2k pra som de qualidade maior do que do hardware.</li>
<li>Suporte pra re-gravação das execuções de assistência das ferramentas.</li>
<li>Suporte do Lua pra scripting.</li>
<li>Uma suíte de debug abrangente.</li>
<li>Suporte pra adaptador wireless.</li>
</ul>

<h2>Plataformas suportadas</h2>

<ul>
<li>Windows 7 ou mais novo</li>
<li>OS X 10.8 (Mountain Lion)<a href="#osxver"><sup>[3]</sup></a> ou mais novo</li>
<li>Linux</li>
<li>FreeBSD</li>
<li>Nintendo 3DS</li>
<li>Nintendo Switch</li>
<li>Wii</li>
<li>PlayStation Vita</li>
</ul>

<p>É conhecido que outras plataformas tipo Unix tais como o OpenBSD funcionam bem mas não são testadas e não são totalmente suportadas.</p>

<h3>Requerimentos do sistema</h3>

> <p>Os requerimentos são mínimos. Qualquer computador que consiga executar o Windows Vista ou mais novo deve ser capaz de lidar com a emulação. Suporte pra OpenGL 1.1 ou mais novo também é requerido, com o OpenGL 3.2 ou mais novo pros shaders e funções avançadas.</p>

Reviewer: ```suggestion
<p>Os requerimentos são mínimos. Qualquer computador que consiga executar o Windows Vista ou mais novo deve ser capaz de lidar com a emulação. Suporte para OpenGL 1.1 ou mais novo também é requerido, com o OpenGL 3.2 ou mais novo para os shaders e funções avançadas.</p>
```

</ul>

<p><a name="flashdetect">[2]</a> A detecção do tamanho da memória flash não funciona em alguns casos. Estes podem ser configurados no runtime mas é recomendado reportar um bug se tal caso for encontrado.</p>

<p><a name="osxver">[3]</a> A 10.8 só é necessária para o porte do Qt. Pode ser possível contruir ou executar o porte do Qt na 10.7 ou mais antiga mas isto não é suportado oficialmente. É conhecido que o porte do SDL funciona na 10.5 e pode funcionar nas mais antigas.</p>

<h2>Copyright</h2>

<p>O mGBA tem um Copyright de © 2013 – 2021 do Jeffrey Pfau. É distribuído sob a <a href="https://www.mozilla.org/MPL/2.0/">Licença Pública do Mozilla, versão 2.0</a>. Uma cópia da licença está disponível no arquivo distribuído da LICENÇA.</p>

<p>O mGBA contém as seguintes bibliotecas de terceiros:</p>

<ul>
<li><a href="https://github.com/benhoyt/inih">inih</a>, a qual tem um copyright de © 2009 – 2020 do Ben Hoyt e é usada sob uma licença de 3 claúsulas do BSD.</li>
<li><a href="https://code.google.com/archive/p/blip-buf">blip-buf</a>, a qual tem um copyright de © 2003 – 2009 do Shay Green e é usada sob uma Licença Pública Inferior do GNU.</li>

> <li><a href="http://www.7-zip.org/sdk.html">SDK do LZMA</a>, a qual é de domínio público.</li>

Reviewer: ```suggestion
<li><a href="http://www.7-zip.org/sdk.html">SDK do LZMA</a>, o qual está sob o domínio público.</li>
```

> <li><a href="https://github.com/aappleby/smhasher">MurmurHash3</a> implementação do Austin Appleby, a qual é de domínio público.</li>

Reviewer: ```suggestion
<li>Implementação da <a href="https://github.com/aappleby/smhasher">MurmurHash3</a> por Austin Appleby, a qual está sob o domínio público.</li>
```

> <li><a href="https://github.com/skandhurkat/Getopt-for-Visual-Studio/">getopt pro MSVC</a>, a qual é de domínio público.</li>

Reviewer: ```suggestion
<li><a href="https://github.com/skandhurkat/Getopt-for-Visual-Studio/">getopt pro MSVC</a>, a qual está sob domínio público.</li>
```

> <li><a href="https://www.sqlite.org">SQLite3</a>, a qual é de domínio público.</li>

Reviewer: ```suggestion
<li><a href="https://www.sqlite.org">SQLite3</a>, a qual está sob o domínio público.</li>
```

</ul>

> <p>Se você é um publicador de jogos e desejar licenciar o mGBA pra uso comercial por favor envie um email pra <a href="mailto:licensing@mgba.io">licensing@mgba.io</a> pra mais informações.</p>

Reviewer: ```suggestion
<p>Se você é um publicador de jogos e desejar licenciar o mGBA para uso comercial por favor envie um email para <a href="mailto:licensing@mgba.io">licensing@mgba.io</a> para mais informações.</p>
```
